airqo-platform/AirQo-frontend
# [Analytics] Terms and conditions page

- What does this PR do?
This PR introduces the terms and conditions page to the Analytics Registrations Page

- What JIRA task is related to this PR?
Shall add the link asap..experiencing a poor internet connection

- How do I test out this PR?
 `cd AirQo-api/analytics`
`npm install`
`npm run local`

- Which end points should are ready for testing?
No endpoints for testing...
